components/layout/blog-post/blog-post.hbs

```diff
@@ -0,0 +1,244 @@
+<div class="section--bg-prim-blue">
+
+    <div class="blog-post-hero section__inner">
+        <h1 class="blog-post-hero__title">If your team isn’t learning,
+            they’re falling behind. Fix it.</h1>
+        <div class="blog-post-hero__blog-author">
+            <div class="blog-post-hero__blog-author-img-container">
+                <img class="blog-post-hero__blog-author-img-container__author-image"
+                    src="http://placekitten.com/300/300" />
+            </div>
+            <div class="blog-post-hero__blog-author-info">
+                <span class="blog-post-hero__blog-author-info-name blog-author-name">By <a href="#">Meow
+                        Murray</a></span>
+                <p class="blog-post-hero__blog-author-info-title">President, Community & Customer Engagement</p>
+            </div><!-- .blog-post-hero__blog-author-info -->
+        </div><!-- .blog-post-hero__blog-author -->
+
+        <div class="blog-post-hero__time-social">
+            <div class="blog-post-hero__time-social-read-time">
+                <p class="blog-post-hero__time-social-read-time--p-text">Time</p>
+                <span class="blog-post-hero__time-social-read-time--p-time">10 minutes</span>
+            </div>
+            <div class="blog-post-hero__time-social-links">
+                <a href="https://www.linkedin.com/company/aquent" title="LinkedIn" target="_blank"
+                    class="social-icon__link">
+                    <img class="social-icon--linkedin" /><span class="show-for-sr">LinkedIn</span>
+                </a>
+                <a href="https://twitter.com/Vitamin T" title="Twitter" target="_blank" class="social-icon__link">
+                    <img class="social-icon--twitter" /><span class="show-for-sr">Twitter</span>
+                </a>
+                <a href="https://www.facebook.com/aquent/" title="Facebook" target="_blank" class="social-icon__link">
+                    <img class="social-icon--facebook" /><span class="show-for-sr">Facebook</span>
+                </a>
+            </div>
+        </div><!-- .blog-post-hero__time-social -->
+    </div><!-- .blog-post-hero .section__inner -->
+
+</div><!-- .section--bg-prim-blue -->
+
+<div class="section--bg-white">
+    <div class="section__inner blog-article">
+        <picture class="blog-article__image-featured">
+            <source media="(max-width: 799px)" srcset="https://www.fillmurray.com/g/460/355">
+            <source media="(min-width: 800px)" srcset="https://www.fillmurray.com/g/1150/646">
+            <img src="https://www.fillmurray.com/g/460/355" role="presentation" alt="" />
+            <source>
+        </picture><!-- .blog-article__image-featured -->
+        <div class="blog-article__post">
+            <p><span class="dropcap">C</span>onsider this scenario: Rebecca is Head of Design at a global
```

Comment:
Instead of creating a new block class that is manually entered into this, could we change it to use a `::first-letter` psuedo-selector so that it is just applied a a course of the content?